src/groups/group.jl

```diff
@@ -266,94 +189,98 @@ Inverse $p^{-1} ∈ \mathcal{G}$ of an element $p ∈ \mathcal{G}$, such that
 $p \circ p^{-1} = p^{-1} \circ p = e ∈ \mathcal{G}$, where $e$ is the [`identity`](@ref)
 element of $\mathcal{G}$.
 """
-inv(M::Manifold, p) = inv(M, p, is_decorator_manifold(M))
-inv(M::Manifold, p, ::Val{true}) = inv(M.manifold, p)
-function inv(M::Manifold, p, ::Val{false})
-    return error("inv not implemented on $(typeof(M)) for points $(typeof(p))")
-end
-function inv(G::AbstractGroupManifold, p)
+inv(M::AbstractGroupManifold, ::Any...)
+@decorator_transparent_function :intransparent function inv(G::AbstractGroupManifold, p)
     q = allocate_result(G, inv, p)
     return inv!(G, q, p)
 end
 
-inv!(M::Manifold, q, p) = inv!(M, q, p, is_decorator_manifold(M))
-inv!(M::Manifold, q, p, ::Val{true}) = inv!(M.manifold, q, p)
-function inv!(M::Manifold, q, p, ::Val{false})
-    return error("inv! not implemented on $(typeof(M)) for points $(typeof(p))")
+@decorator_transparent_function :intransparent function inv!(M::AbstractGroupManifold, q, p)
+    inv!(M.manifold, q, p)
 end
-
```

Comment:
By convention, newlines should be between definitions of different functions, including between mutating and non-mutating variants, but no newlines should be between definitions of different methods of the same function. Here and below.

Reply:
I unified that.

Currently also the specification which transparency status a certain implementation has is given where the function is defined. This is agains the idea of ordering alphabetically but I think it's helpful to keep such information where the function is defined.